tests/e2e/loadbalancer.go

```diff
@@ -400,3 +422,97 @@ func getLBTargetCount(ctx context.Context, lbDNSName string, expectedTargets int
 	}
 	return nil
 }
+
+// testClusterConfig handle information for cloud-config configuration in the running cluster.
+type testClusterConfig struct {
+	kubeClient          clientset.Interface
+	configMapName       string
+	configMapNamespace  string
+	configMapKey        string
+	originalConfig      *v1.ConfigMap
+	originalConfigValue string
+}
+
+var supportedConfigLookup = []string{"openshift-config/cloud-provider-config/config"}
```

Comment:
TODO we need to figure out what is the config on other distributions, starting from tested on CI